tket2-py/src/utils.rs

```diff
@@ -45,9 +56,51 @@ macro_rules! create_py_exception {
     };
 }
 pub(crate) use create_py_exception;
-use itertools::Itertools;
 
 /// Convert an iterator of one type into vector of another type.
 pub fn into_vec<T, S: From<T>>(v: impl IntoIterator<Item = T>) -> Vec<S> {
     v.into_iter().map_into().collect()
 }
+
+/// Utility for building a module with a single circuit definition.
+#[allow(unused)]
```

Comment:
What's unused?

Reply:
It's only used when testing, so It may get reported as unused on non-test builds.
I copied this from `tket2`, but maybe it should go inside the `test` module.